.github/workflows/pr-add-team-labels.yml

```diff
@@ -0,0 +1,23 @@
+name: Add team labels
+on:
+  pull_request:
+    types:
+      - review_requested
+
+jobs:
+  add_team_label:
+    runs-on: ubuntu-latest
+    name: Add team label
+    permissions:
+      pull-requests: write
+    steps:
+      - name: Add labels
+        run: |
+          for team in $(jq -r '.event.pull_request.requested_teams[].name' <<< "$GITHUB_CONTEXT"); do 
+            gh pr edit $NUMBER --add-label "team/$team"
+          done
+        env:
+          GITHUB_CONTEXT: ${{ toJson(github) }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Is this token linked to a user account or service account? Trying to ensure that the action continues to operate even with change in team members. If it is linked to user account, let's add a comment clarifying the things to do if the token gets invalidated.

Reply:
No need to worry about that https://docs.github.com/en/actions/security-guides/automatic-token-authentication#about-the-github_token-secret